integreat_cms/static/src/js/bulk-actions.ts

```diff
@@ -95,16 +82,35 @@ window.addEventListener("load", () => {
     form.submit();
   }
 
-  function hasTranslation(): boolean {
-    // checks if at least one of the selected pages has a translation for the current language
-    let languageSlug = document.getElementById("pdf-export-option").dataset.languageSlug;
-    return selectItems
+});
+
+export function toggleBulkActionButton() {
+  // Only activate button if at least one item and the action is selected
+  // also check if at least one page translation exists for PDF export before activation
+  const selectItems = Array.from(document.getElementsByClassName("bulk-select-item"));
+  let selectedAction = bulkAction.options[bulkAction.selectedIndex];
+  if (
+    !selectItems
       .filter(isInputElement)
-      .filter(inputElement => inputElement.checked)
-      .some(inputElement => {
-        return inputElement
-          .closest("tr")
-          .querySelector(`.lang-grid .${languageSlug} .no-trans`) === null;
-      });
+      .some((el) => el.checked) ||
+    bulkAction.selectedIndex === 0 ||
+    (selectedAction.id === "pdf-export-option" && !hasTranslation(selectItems))
```

Comment:
Instead of hardcoding the option id here, I'd suggest to add a custom data attribute, e.g. `data-require-existing-translation` or similar, which could then be reused for other bulk actions in the future which also might require a translation.

Reply:
I think the "files changed" tab is a bit misleading here :joy: 
Actually all I did was moving 2 function (toggleBulkActionButton and hasTanslation) out of the inner load window scope to apply the export expression, because 'import' and 'export' may only appear at the top level
But sure we can take the opportunity to improve here a bit

Reply:
Oh ok, sorry didn't notice :see_no_evil: 
Well then let's leave it as it is, we can still do this once we have another bulk action which requires it^^